store/copr/region_cache.go

```diff
@@ -67,67 +68,84 @@ func (l *LocationKeyRanges) getBucketVersion() uint64 {
 	return l.Location.GetBucketVersion()
 }
 
+// getKeyLocation used in splitKeyRangeByBuckets Only
+func getKeyLocation(l *LocationKeyRanges) *tikv.KeyLocation {
+	// Sometimes, there may be some delay between region information and buckets information within the region.
+	// It may cause some gaps between the region startKey and first buckets key and region endKey and the last buckets key.
+	// To easy the implementation of splitKeyRangeByBuckets, we copy the l.Location (with original value untouched) and add region startKey
```

Comment:
To ease the implementation? Some tools like Grammarly could help reduce grammatical mistakes.